pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -906,7 +906,7 @@ func (gce *GCECloud) projectOwnsStaticIP(name, region string, ipAddress string)
 	return false, nil
 }
 
-func (gce *GCECloud) ensureStaticIP(name, serviceName, region, existingIP string) (ipAddress string, created bool, err error) {
+func ensureStaticIP(s CloudAddressService, name, serviceName, region, existingIP string) (ipAddress string, existing bool, err error) {
```

Comment:
Passing the interface via parameter is a temporary, ugly plug. One topic of discussion for us is should we separate our wrapper GCP API funcs from LB-specific funcs?